example/app.py

```diff
@@ -22,7 +22,7 @@
 import flask
 from flask import redirect, request, jsonify, make_response, render_template, session, url_for
 from flask import Flask
-from flask_jsonlocale import Locales
+from flask_jsonlocaleBubblySprout import Locales
```

Comment:
why?

Reply:
Oops, sorry. I created a separate file when I was testing and I had to link it to that one.